module/Endjin.ZeroFailed.Build.psd1

```diff
@@ -0,0 +1,141 @@
+# <copyright file="Endjin.ZeroFailed.Build.psd1" company="Endjin Limited">
+# Copyright (c) Endjin Limited. All rights reserved.
+# </copyright>
+@{
+
+# Script module or binary module file associated with this manifest.
+RootModule = 'Endjin.ZeroFailed.Build.psm1'
+
+# Version number of this module.
+ModuleVersion = '0.0.1'
+
+# Supported PSEditions
+CompatiblePSEditions = @("Core")
+
+# ID used to uniquely identify this module
+GUID = '27d44114-63c9-4237-ae51-05e3a80e0b6a'
+
+# Author of this module
+Author = 'Endjineers'
+
+# Company or vendor of this module
+CompanyName = 'Endjin Limited'
+
+# Copyright statement for this module
+Copyright = '(c) endjin. All rights reserved.'
+
+# Description of the functionality provided by this module
+Description = 'Provides Endjin-specific extensions for use with ZeroFailed-based build processes'
+
+# Minimum version of the PowerShell engine required by this module
+PowerShellVersion = '7.0'
+
+# Name of the PowerShell host required by this module
+# PowerShellHostName = ''
+
+# Minimum version of the PowerShell host required by this module
+# PowerShellHostVersion = ''
+
+# Minimum version of Microsoft .NET Framework required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# DotNetFrameworkVersion = ''
+
+# Minimum version of the common language runtime (CLR) required by this module. This prerequisite is valid for the PowerShell Desktop edition only.
+# ClrVersion = ''
+
+# Processor architecture (None, X86, Amd64) required by this module
+# ProcessorArchitecture = ''
+
+# Modules that must be imported into the global environment prior to importing this module
+# RequiredModules = @()
+
+# Assemblies that must be loaded prior to importing this module
+# RequiredAssemblies = @()
+
+# Script files (.ps1) that are run in the caller's environment prior to importing this module.
+# ScriptsToProcess = @()
+
+# Type files (.ps1xml) to be loaded when importing this module
+# TypesToProcess = @()
+
+# Format files (.ps1xml) to be loaded when importing this module
+# FormatsToProcess = @()
+
+# Modules to import as nested modules of the module specified in RootModule/ModuleToProcess
+# NestedModules = @()
+
+# Functions to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no functions to export.
+# FunctionsToExport = @()
+
+# Cmdlets to export from this module, for best performance, do not use wildcards and do not delete the entry, use an empty array if there are no cmdlets to export.
+CmdletsToExport = @()
+
+# Variables to export from this module
+VariablesToExport = '*'

```

Comment:
Now that these are smaller units, should we be explicit about the variables in module/tasks/codeops.properties.ps1 ?

Reply:
None of the variables are actually exported since we are doing an explicit `Export-ModuleMember` in the `.psm1` file.

Updated to make this more obvious.